kde-misc/corectrl/corectrl-9999.ebuild

```diff
@@ -22,16 +22,13 @@ IUSE="+pcinames"
 REQUIRED_USE=""
 
 DEPEND="
-	dev-qt/qtcore:5
-	dev-qt/qtcharts:5[qml]
-	dev-qt/qtdbus:5
-	dev-qt/qtnetwork:5
-	dev-qt/qtquickcontrols2:5
-	dev-qt/qtsvg:5
-	dev-qt/qtwidgets:5
-	dev-qt/linguist-tools:5[qml]
+	dev-qt/qtbase:6[dbus,network,widgets]
+	dev-qt/qtcharts:6[qml]
+	dev-qt/qtdeclarative:6
+	dev-qt/qtsvg:6
+	dev-qt/qttools:6[linguist]
```

Comment:
could you add qml use flag here too or is it no longer needed?

Reply:
Sorry, i don't know which other package would need "qml" use flag, apart from qtcharts.

I followed this: https://wiki.gentoo.org/wiki/Project:Qt/Qt6_migration_notes

Reply:
qttools also has the qml flag which is where i put my comment

Reply:
I'm sorry for the delay in updating and adding the qml use flag, was busy with work. But it's now added, if you want to merge.